### PR TITLE
initial DD1299 ui (submit button doesn't send to server)

### PR DESCRIPTION
**Pull request by tinyels**

### CODE REVIEW

File: src/scenes/DD1299/DD1299Form.jsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+   textbox vs textarea (e.g for addresses)
+   dates look wonky
+   styling in accordance with USWDS
+   formatting of labels?
+  */
+  const field = fields[name];
+  if (field.type !== 'group') return renderField(name, fields);
+  return (
+    <fieldset key={name}>
+      <legend htmlFor={name}>{field.label}</legend>
```
Comment (ntwyman) [inline src/scenes/DD1299/DD1299Form.jsx]: I think you want field.title not field.label. The title of groups isn't showing up.

File: src/scenes/DD1299/fields.js [hunk 2]
```diff
@@ -0,0 +1,276 @@
...
+            example: Roll
+            title: Last Name
+          service_member_rank:
+            type: string
+            example: Commodore
+            title: Rank/Grade
+          service_member_ssn:
+            type: string
+            pattern: '^\\d{3}-\\d{2}-\\d{4}$'
+            example: 555-555-5555
```
Comment (ntwyman) [inline src/scenes/DD1299/fields.js]: 555-55-5555

File: src/scenes/DD1299/fields.js [hunk 3]
```diff
@@ -0,0 +1,276 @@
...
+            title: First Name
+          service_member_middle_initial:
+            example: L.
+            nullable: true
+            title: Middle Initial
+          service_member_last_name:
+            type: string
+            example: Roll
+            title: Last Name
+          service_member_rank:
```
Comment (ntwyman) [inline src/scenes/DD1299/fields.js]: This should probably be a drop down?

File: src/scenes/DD1299/fields.js (hunk 2 above)
Comment (tinyels) [reply src/scenes/DD1299/fields.js]: I really should remove this pull request--got incoming changes that are fairly different.

**Pull request closed without merging**